lib/ansible/modules/cloud/vmware/vmware_dvswitch.py

```diff
@@ -201,6 +209,7 @@ def main():
     argument_spec.update(dict(datacenter_name=dict(required=True, type='str'),
                               switch_name=dict(required=True, type='str'),
                               mtu=dict(required=True, type='int'),
+                              lacp=dict(required=True, type='str'),
```

Comment:
Could you set this to default=False? LACP shouldn't be enabled by default.

Reply:
This is also the reason why the test case "TASK [vmware_dvswitch : add distributed vSwitch]" fails. Maybe you should also try to create an additional test for LACP. Not sure if you this works with vcsim though..

Reply:
I have configured LACP to True because in my mockup, when I created a DVS, the LACP menu was not available. I had to right click on DVS to upgrade for LACP support. 
With the LACP option configured on True, the LACP menu is available. 
But today, it fails because of a vanity validate-module error in shippable.